tests/cephfs/snapshot_clone/clone_states.py

```diff
@@ -0,0 +1,129 @@
+import logging
```

Comment:
I think we are covering this scenario in clone_status.py test. Can we remove this test?

Reply:
With the above changes suggested i think we are checking transition flow as part if clone_states and clone_status covers different platforms(fuse,Kernel)

Reply:
I think we can just add the transition flow in clone_status which will cover both the scenario of clone_states & both platforms(fuse,kernel) And we can remove clone_states.